app/validations/test_payload_xunit_parser.rb

```diff
@@ -136,7 +136,7 @@ def parse_test_suite suite, parsed
 
       # parse the "name" attribute of the <testcase> (mandatory)
       check_string_attribute test_case, :name, limit: false do |value|
-        normalized_value = value.strip
+        normalized_value = value.strip.underscore.humanize
```

Comment:
We should only do this when the name contains no spaces, so that we don't modify already human-friendly test names coming from Jasmine, RSpec, etc.

**ALSO** modify the migration to take this into account.